lib/util.py

```diff
@@ -486,7 +465,7 @@ def block_explorer_URL(config, kind, item):
     if not kind_str:
         return
     url_parts = [be_tuple[0], kind_str, item]
-    return ''.join(url_parts)
+    return "/".join(url_parts)
```

Comment:
These parts already have "/" suffix. Do you really want to join with another "/"?

Reply:
:/ how did this made its way into the code? Anyway, i'll remove it :D

Reply:
Done

Reply:
Please test this before the release

Reply:
This has been reverted here https://github.com/BTCGPU/electrum/pull/6/commits/885ab6bb4954396f26d565e27ad50918dd8fb9c1